react-common/styles/controls/Checkbox.less

```diff
@@ -1,6 +1,6 @@
 .common-checkbox {
     position: relative;
-    padding: 0.25rem;
+    // padding: 0.25rem;
```

Comment:
Does the checkbox scale when zoomed in? I think it should, but just curious.

Reply:
Yes, it scales with zoom.